### PR TITLE
Add Login and Signup UI changes and Prevent unauthenticated user from using some commands

### LINKED ISSUE #31: As a manager, i want to login/signup of the application
So that i can login, logout and signup from the application

For V1.2, 
- [x] Update UI (logged in and logged out screens)
- [x] Add System test
- [x] Store user login as session

### PR DESCRIPTION
**Pull request by kohchihao**
In this PR,
- Update UI when user login 
- Update UI after user signup
- Prevent unauthenticated user from using all the `/order` command
- Unauthenticated user can use `/login` , `/signup` , `/help` , `/history`
- Added the necessary test for relevant classes

Fixes #31

### CODE REVIEW

File: src/test/java/systemtests/AddCommandSystemTest.java [hunk 1]
```diff
@@ -48,14 +53,20 @@
     public void add() {
         Model model = getModel();
 
+        /* Login */
+        String loginCommand = LoginCommand.COMMAND_WORD + " ";
+        String command = loginCommand + PREFIX_USERNAME + VALID_MANAGER_USERNAME_ALICE
+                + " " + PREFIX_PASSWORD + VALID_MANAGER_PASSWORD_ALICE;
+        executeCommand(command);
+
```
Comment (jinyingtan) [inline src/test/java/systemtests/AddCommandSystemTest.java]: Could consider adding an assertCommandSuccess to make sure that the command is successful

File: src/test/java/systemtests/HelpCommandSystemTest.java [hunk 2]
```diff
@@ -31,6 +36,13 @@
 
     @Test
     public void openHelpWindow() {
+
```
Comment (jinyingtan) [inline src/test/java/systemtests/HelpCommandSystemTest.java]: Add a assertSuccess to check if help window open without the need to login

Comment (kohchihao) [reply src/test/java/systemtests/HelpCommandSystemTest.java]: Resolved inside [151829b](https://github.com/CS2103-AY1819S1-T16-3/main/pull/52/commits/151829b89501efcafd8b314dbf6032764fb69cb2)

**Comment by kohchihao** [comment]
Fixes #65

**Pull request closed without merging**